c-bindings/include/geode/region.h

```diff
@@ -40,18 +40,41 @@ APACHE_GEODE_C_EXPORT void apache_geode_Region_PutString(
     apache_geode_region_t* region, const char* key, const char* value);
 
 APACHE_GEODE_C_EXPORT void apache_geode_Region_PutByteArray(
-    apache_geode_region_t* region, const char* key, const char* value, size_t size);
+    apache_geode_region_t* region, const char* key, size_t keyLength,
```

Comment:
We should really require length parameters for all array pointers, it's not safe to rely on `NULL` termination.

Reply:
I don't disagree. However, I'm thinking we should consider the single more powerful apache_geode_Region_Put (line 50), which includes lengths for both key and value, and do away with all other versions of the Put API.